server/src/main/java/com/google/coffeehouse/storagehandler/StorageHandler.java

```diff
@@ -71,7 +71,7 @@ public static Person getPerson(DatabaseClient dbClient, String userId) {
                                            .setNickname(row.getString(/* nicknameIndex= */1))
                                            .setUserId(userId);
 
-      if (!row.isNull(/* index= */2) || !row.getString(/* index= */2).isEmpty()) {
+      if (!row.isNull(/* index= */2)) {
```

Comment:
Thanks for catching this issue! Checking if it is empty is honestly useless, since this could be nullable in the DB.